### ISSUE TITLE
Hint text padding not aligning to the left

### ISSUE THREAD

**Issue by azizj1** [ganfra/MaterialSpinner#19]
I made a custom adapter layout in an attempt to left align the selected texts AND the hinted text:

```
    String[] ITEMS = {"Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6"};
    MyAdapter adapter = new MyAdapter(this, R.layout.spinner_item_without_padding, ITEMS);
    adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
    MaterialSpinner spinner = (MaterialSpinner) findViewById(R.id.accountSpinner);
    spinner.setAdapter(adapter);
```

where spinner_item_without_padding is:

<?xml version="1.0" encoding="utf-8"?>
<TextView xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content" android:textSize="16sp" android:textColor="@color/black_text">

</TextView>

The selected text gets aligned properly to the left, but the hinted text is still misaligned by 6dp-ish. 

Thank you and I look forward to your reply!

**Comment by ganfra** [comment]
I could add this feature to left align ... actually, I don't use the same layout for the hint that the ones provided to the adapter !

**Comment by mdumrauf** [comment]
Fixed with #64
